react/features/base/jwt/middleware.ts

```diff
@@ -153,6 +153,12 @@ function _setJWT(store: IStore, next: Function, action: AnyAction) {
                     _overwriteLocalParticipant(
                         store, { ...newUser,
                             features: context.features });
+                } else if (jwtPayload.name || jwtPayload.picture) {
```

Comment:
Just verifying: we will have either context or either `picture` and `name but never both. is this correct?

Reply:
both ...